src/main/java/in/projecteka/consentmanager/user/OtpAttemptRepository.java

```diff
@@ -0,0 +1,56 @@
+package in.projecteka.consentmanager.user;
+
+import in.projecteka.consentmanager.common.DbOperationError;
+import in.projecteka.consentmanager.user.model.OtpAttempt;
+import io.vertx.pgclient.PgPool;
+import io.vertx.sqlclient.Tuple;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Repository;
+import reactor.core.publisher.Mono;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+
+@Repository
+@AllArgsConstructor
+public class OtpAttemptRepository {
+
+    private static final String INSERT_OTP_ATTEMPT = "INSERT INTO " +
+            "otp_attempt (phone_number,blocked) VALUES ($1,$2)";
+
+    private static final String SELECT_OTP_ATTEMPT = "SELECT * FROM otp_attempt " +
+            "WHERE phone_number = $1 " +
+            "ORDER BY timestamp DESC LIMIT $2";
```

Comment:
never use column name as `timestamp`.

Reply:
Ok. We will rename it.

Reply:
We named it timestamp because we are using `timestamp` as  a column name in everywhere else.